app/components/nextflow/samplesheet/metadata_cell_component.rb

```diff
@@ -4,17 +4,17 @@ module Nextflow
   module Samplesheet
     # Component to render a cell with sample metadata into the sample sheet
     class MetadataCellComponent < ViewComponent::Base
-      attr_reader :form, :metadata, :name
+      attr_reader :form, :metadata, :name, :sample
 
       def initialize(form:, name:, sample:, entry:)
         @form = form
         @name = name
-        @metadata = sample_metadata(entry, sample)
+        @sample = sample
+        @metadata = metadata_value(sample, entry)
       end
 
-      def sample_metadata(entry, sample)
-        key = entry['meta'].is_a?(Array) ? entry['meta'][0].to_s : entry['meta'].to_s
-        sample.metadata.key?(key) ? sample.metadata[key] : ''
+      def metadata_value(sample, _entry)
```

Comment:
Looks like `entry` is not being used here so you can probably remove it from the method signature and any calls to it

Reply:
Updated in f48755888474d4ee63d817004a4041a84b2ae683